src/components/rocket/index.js

```diff
@@ -8,54 +11,47 @@ import {
   ImageHolder,
   ReserveBtn,
   Inner,
+  Loading,
+  Group,
+  Mark,
 } from './styles/rocket';
 
 export default function Rocket() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
```

Comment:
It's best practice to avoid making a request to the API on the page rerender, kindly only dispatch the fetch action if the state is empty.